pynxtools_stm/nomad/entrypoints.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Entry points for STS and STM examples."""
+
+from nomad.config.models.plugins import ExampleUploadEntryPoint
+
+sts_example = ExampleUploadEntryPoint(
+    title="Scanning Tunneling Spectroscopy (STS)",
+    category="SPM experiments examples",
+    description="""This example presents the capabilities of the NOMAD platform to store standarized Scanning Tunneling Spectroscopy (sts) data.""",
+    path="pynxtools-stm/nomad/examples",
+    local_path="examples/data/sts",
```

Comment:
This should be a `.zip`. But, I am currently testing if we actually need this, so maybe this can also be removed.